[MST-543] Allow InstructorService to retrieve proctoring escalation email

bseverino opened the pull request:
[MST-543](https://openedx.atlassian.net/browse/MST-543)

This allows edx-proctoring to access the proctoring escalation email address for a given course.

schenedx reviewed (approved):
👍

matthugs commented on cms/djangoapps/contentstore/services.py:
```diff
@@ -0,0 +1,35 @@
+"""
```
Do we want this in the `cms/djangoapps` folder? I expect the automatically generated "suspicious" emails are generated on the LMS side; maybe we should put this in a shared folder?

bseverino replied:
Hmm that's a good point, let me see if there's a better spot for it

bseverino commented:
Updated to leverage the existing instructor service rather than creating a new service, and updated https://github.com/edx/edx-proctoring/pull/754 to account for these changes.
